gradle.properties

```diff
@@ -2,16 +2,16 @@
 org.gradle.jvmargs=-Xmx1G
 # Fabric Properties
 # check these on https://modmuss50.me/fabric.html
-minecraft_version=1.19.3
-yarn_mappings=1.19.3+build.4
-loader_version=0.14.12
+minecraft_version=1.19.4
+yarn_mappings=1.19.4+build.2
+loader_version=0.14.19
 
 # Mod Properties
 mod_id=dsurround
-mod_version=0.1.1
+mod_version=0.1.1.a
```

Comment:
This is not correct. For now, main (1.19.2 mc version) has 0.1.1 version, 1.19.3 branch will has 0.1.2 version
Respectively, 1.19.4 branch will has 0.1.3 version